presto-native-execution/CMakeLists.txt

```diff
@@ -155,12 +155,6 @@ resolve_dependency(glog)
 set_source(fmt)
 resolve_dependency(fmt)
 
-# TODO
-find_library(EVENT event)
-
-# TODO
-find_library(DOUBLE_CONVERSION double-conversion)
```

Comment:
event and double-conversion appear to be not needed at all - I can build locally without them